Unify the style of the TWC articles (#2178)
ppy/osu-wiki: wiki/Tournaments/TWC/2014/en.md

---
tags:

> - TWC2014

Reviewer: ```suggestion
- TWC 2014
- TWC2014
```

- TWC 2014
- TWC2014
---
# osu! Taiko World Cup 2014

![TWC 2014 Logo](img/logo.png)

> The **osu! Taiko World Cup 2014** (***TWC 2014***) was a country-based Taiko tournament hosted by the [osu! staff](/wiki/People/The_Team). It was the 4th installment of the osu!taiko World Cup.

Reviewer 2: ```suggestion
The **osu! Taiko World Cup 2014** (***TWC 2014***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the fourth installment of the osu!taiko World Cup.
```

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2014-01-16/2014-01-26 |
| Live Drawings | 2014-02-01 (13:00 UTC) |
| Group Stage | 2014-02-07/2014-02-09 |
| Round of 16 | 2014-02-15/2014-02-16 |
| Quarterfinals | 2014-02-22/2014-02-23 |
| Semifinals | 2014-03-01/2014-03-02 |
| Finals | 2014-03-08/2014-03-09 |
| Grand Finals | 2014-03-15 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, unique profile badge, TWC trophy |

Author: the unique one?

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, unique profile badge, TWC trophy |

Reviewer 3: No problems with it.

| ![][flag_SG] | **Singapore** | **[CoroQuetz](https://osu.ppy.sh/users/499843)**, [minray](https://osu.ppy.sh/users/653771), [neonat](https://osu.ppy.sh/users/1561995), [DJNightmare](https://osu.ppy.sh/users/70909) |
| ![][flag_KR] | **South Korea** | **[Konpaku Sariel](https://osu.ppy.sh/users/533502)**, [eoe](https://osu.ppy.sh/users/eoe), [-applerss-](https://osu.ppy.sh/users/983349), [L y s](https://osu.ppy.sh/users/211825), [\[Akita-Neru\]](https://osu.ppy.sh/users/613944) |
| ![][flag_ES] | **Spain** | **[LZD](https://osu.ppy.sh/users/224335)**, [MYRKUL](https://osu.ppy.sh/users/255933), [CARRI](https://osu.ppy.sh/users/854536), [Hanjamon](https://osu.ppy.sh/users/1703330), [son\_gohan\_2](https://osu.ppy.sh/users/443305), [Ares267](https://osu.ppy.sh/users/174378) |
| ![][flag_SE] | **Sweden** | **[boat](https://osu.ppy.sh/users/276074)**, [Tanaka](https://osu.ppy.sh/users/1897905), [birrin](https://osu.ppy.sh/users/2290294), [Fuwuwi](https://osu.ppy.sh/users/88200) |
| ![][flag_UA] | **Ukraine** | **[Pagliaccio](https://osu.ppy.sh/users/1163187)**, [Anny](https://osu.ppy.sh/users/249094), [Aka](https://osu.ppy.sh/users/1307553) |

> | ![][flag_GB] | **United Kingdom** | **[xWillx](https://osu.ppy.sh/users/696222)**, [abrian159](https://osu.ppy.sh/users/1159051), [Dainesl](https://osu.ppy.sh/users/Dainesl), [TurnipLover](https://osu.ppy.sh/users/TurnipLover), [Lno](https://osu.ppy.sh/users/1900787) |

Reviewer 3: ```suggestion
| ![][flag_GB] | **United Kingdom** | **[xWillx](https://osu.ppy.sh/users/696222)**, [abrian159](https://osu.ppy.sh/users/1159051), [Dainesl](https://osu.ppy.sh/users/1588551), [TurnipLover](https://osu.ppy.sh/users/TurnipLover), [Lno](https://osu.ppy.sh/users/1900787) |
```

| Management | ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703) |
| Map Selector | ![][flag_DE] [OnosakiHito](https://osu.ppy.sh/users/290128) |
| Streamer | ![][flag_FR] [shARPII](https://osu.ppy.sh/users/776257) |
| Commentators | ![][flag_FR] [Mr Color](https://osu.ppy.sh/users/116078), ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370), ![][flag_DE] [OnosakiHito](https://osu.ppy.sh/users/290128), ![][flag_US] [ztrot](https://osu.ppy.sh/users/6347), ![][flag_NO] [Dolphin](https://osu.ppy.sh/users/1331438) |
| Statistician | ![][flag_PL] [Marcin](https://osu.ppy.sh/users/722665) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/179127)
- [Livestream](https://www.twitch.tv/osulive/)

-----------------------

## Participants

|  | Country | Members |
| :-: | :-: | :-- |
| ![][flag_AR] | **Argentina** | **[whosthebox](https://osu.ppy.sh/u/1451311)**, [Nearex](https://osu.ppy.sh/users/1893238), [Pollotuc](https://osu.ppy.sh/users/42440), [Chillet](https://osu.ppy.sh/users/1236632), [cyntuu mirandaa](https://osu.ppy.sh/users/724230), [Lost The Lights](https://osu.ppy.sh/users/1255920) |

> | ![][flag_BR] | **Brazil** | **[Kolbe](https://osu.ppy.sh/users/1254286)**, [Yuki-chan](https://osu.ppy.sh/users/93687), - T i g e r -, [suyama-chan](https://osu.ppy.sh/users/928298), [lubask](https://osu.ppy.sh/users/1897332), Wolf |

Reviewer: ```suggestion
| ![][flag_BR] | **Brazil** | **[Kolbe](https://osu.ppy.sh/users/1254286)**, [Yuki-chan](https://osu.ppy.sh/users/93687), [- T i g e r -](https://osu.ppy.sh/users/1381781), [suyama-chan](https://osu.ppy.sh/users/928298), [lubask](https://osu.ppy.sh/users/1897332), [Wolf](https://osu.ppy.sh/users/555855) |
```

| ![][flag_CL] | **Chile** | **[-Anhedonia-](https://osu.ppy.sh/users/920861)**, [megadaisukeone](https://osu.ppy.sh/users/1577133), [Gonzaalooo](https://osu.ppy.sh/users/1948903), [MikuWhiteRockShooter](https://osu.ppy.sh/users/1619547) |
| ![][flag_CN] | **China** | **[- Yuuka Niya -](https://osu.ppy.sh/users/1892059)**, [yys428](https://osu.ppy.sh/users/1061175), [HM495](https://osu.ppy.sh/users/810813), [- Kagami Yuki -](https://osu.ppy.sh/users/1319686), [mousegoomba](https://osu.ppy.sh/users/1035291), [moe moe moe](https://osu.ppy.sh/users/1804904) |
| ![][flag_FR] | **France** | **[\_Gezo\_](https://osu.ppy.sh/users/481582)**, [Nashmun](https://osu.ppy.sh/users/49031), [TimmyAkmed](https://osu.ppy.sh/users/1799973), [Jinyan](https://osu.ppy.sh/users/669531), [Fraolinch](https://osu.ppy.sh/users/205257), [omegaflo](https://osu.ppy.sh/users/83291) |
| ![][flag_DE] | **Germany** | **[Luna](https://osu.ppy.sh/users/588007)**, [tiamus](https://osu.ppy.sh/users/515277), [BrokenArrow](https://osu.ppy.sh/users/384034), [DarkDunskin](https://osu.ppy.sh/users/98985), [\[TaikoTori\]](https://osu.ppy.sh/users/1074143), [Mew104](https://osu.ppy.sh/users/2345156) |

> | ![][flag_HK] | **Hong Kong** | **[thomas1195](https://osu.ppy.sh/users/1691488)**, Andrew Chow, [\[yuyuko\_Saigyouj\]](https://osu.ppy.sh/users/1378994), [ice man007](https://osu.ppy.sh/users/1926937), [Jikson\[m9\]](https://osu.ppy.sh/users/1485257), [aabc271](https://osu.ppy.sh/users/155707) |

Reviewer: ```suggestion
| ![][flag_HK] | **Hong Kong** | **[thomas1195](https://osu.ppy.sh/users/1691488)**, [Andrew Chow](https://osu.ppy.sh/users/3043041), [\[yuyuko\_Saigyouj\]](https://osu.ppy.sh/users/1378994), [ice man007](https://osu.ppy.sh/users/1926937), [Jikson\[m9\]](https://osu.ppy.sh/users/1485257), [aabc271](https://osu.ppy.sh/users/155707) |
```

| ![][flag_PH] | **Philippines** | **[Primrea](https://osu.ppy.sh/users/943743)**, [senkai](https://osu.ppy.sh/users/1150178), [Pizzicato](https://osu.ppy.sh/users/692610), [Kotaro](https://osu.ppy.sh/users/796530) |
| ![][flag_PL] | **Poland** | **[Acrith](https://osu.ppy.sh/users/389880)**, [DarkStoorM](https://osu.ppy.sh/users/174347), [Minkodon](https://osu.ppy.sh/users/1385227), [LZMieczu](https://osu.ppy.sh/users/1797498), [Sylwia132](https://osu.ppy.sh/users/2037319), [IsonamiPL](https://osu.ppy.sh/users/1753234) |
| ![][flag_RU] | **Russian Federation** | **[MarisaLover](https://osu.ppy.sh/users/1359539)**, [Livius777](https://osu.ppy.sh/users/1282662), [magax](https://osu.ppy.sh/users/918645) |
| ![][flag_SG] | **Singapore** | **[CoroQuetz](https://osu.ppy.sh/users/499843)**, [minray](https://osu.ppy.sh/users/653771), [neonat](https://osu.ppy.sh/users/1561995), [DJNightmare](https://osu.ppy.sh/users/70909) |

> | ![][flag_KR] | **South Korea** | **[Konpaku Sariel](https://osu.ppy.sh/users/533502)**, eoe, [-applerss-](https://osu.ppy.sh/users/983349), [L y s](https://osu.ppy.sh/users/211825), [\[Akita-Neru\]](https://osu.ppy.sh/users/613944) |

Reviewer: ```suggestion
| ![][flag_KR] | **South Korea** | **[Konpaku Sariel](https://osu.ppy.sh/users/533502)**, [eoe](https://osu.ppy.sh/users/659959), [-applerss-](https://osu.ppy.sh/users/983349), [L y s](https://osu.ppy.sh/users/211825), [\[Akita-Neru\]](https://osu.ppy.sh/users/613944) |
```

| ![][flag_ES] | **Spain** | **[LZD](https://osu.ppy.sh/users/224335)**, [MYRKUL](https://osu.ppy.sh/users/255933), [CARRI](https://osu.ppy.sh/users/854536), [Hanjamon](https://osu.ppy.sh/users/1703330), [son\_gohan\_2](https://osu.ppy.sh/users/443305), [Ares267](https://osu.ppy.sh/users/174378) |
| ![][flag_SE] | **Sweden** | **[boat](https://osu.ppy.sh/users/276074)**, [Tanaka](https://osu.ppy.sh/users/1897905), [birrin](https://osu.ppy.sh/users/2290294), [Fuwuwi](https://osu.ppy.sh/users/88200) |
| ![][flag_UA] | **Ukraine** | **[Pagliaccio](https://osu.ppy.sh/users/1163187)**, [Anny](https://osu.ppy.sh/users/249094), [Aka](https://osu.ppy.sh/users/1307553) |

> | ![][flag_GB] | **United Kingdom** | **[xWillx](https://osu.ppy.sh/users/696222)**, [abrian159](https://osu.ppy.sh/users/1159051), [Dainesl](https://osu.ppy.sh/users/1588551), TurnipLover, [Lno](https://osu.ppy.sh/users/1900787) |

Reviewer: ```suggestion
| ![][flag_GB] | **United Kingdom** | **[xWillx](https://osu.ppy.sh/users/696222)**, [abrian159](https://osu.ppy.sh/users/1159051), [Dainesl](https://osu.ppy.sh/users/1588551), [TurnipLover](https://osu.ppy.sh/users/1133830), [Lno](https://osu.ppy.sh/users/1900787) |
```

  - [Yucha-P - Thieves Night Trick (Fraolinch) \[Taiko Trick\]](https://osu.ppy.sh/beatmapsets/148785#taiko/367777)
  - [Sta - Iris (MMzz) \[Oni\]](https://osu.ppy.sh/beatmapsets/106598#taiko/279828)
- DoubleTime
  - [Tsubaki - Aijou Pandemic (Nymph) \[KS's Taiko\]](https://osu.ppy.sh/beatmapsets/40166#taiko/129682)
  - [ave;new feat. Sakura Saori - My Sweet Lady (Short Ver.) (osuplayer111) \[Taiko Oni\]](https://osu.ppy.sh/beatmapsets/41209#taiko/131073)
  - [NAOKI feat. SMiLE.dk - A Geisha's Dream (yongtw123) \[Taiko Oni\]](https://osu.ppy.sh/beatmapsets/28766#taiko/103398)
  - [IOSYS - Endless Tewi-ma Park (Kurosanyan) \[Taiko Oni\]](https://osu.ppy.sh/beatmapsets/48874#taiko/153393)
- FreeMod
  - [kors k feat. Furukawa Mirin - Drive Me Crazy (ampzz) \[Kurosanyan's Oni\]](https://osu.ppy.sh/beatmapsets/65950#taiko/215497)
  - [Mutsuhiko Izumi - Red Goose (lepidopodus) \[Oni\]](https://osu.ppy.sh/beatmapsets/55920#taiko/169371)
  - [IOSYS - RE:Usatei (OzzyOzrock) \[Taiko Oni\]](https://osu.ppy.sh/beatmapsets/31343#taiko/103022)
  - [capsule - JUMPER (Mafiamaster) \[Roko-Don's Taiko\]](https://osu.ppy.sh/beatmapsets/21130#taiko/74040)
- Tiebreaker

>   - **[Demetori - Higan Retour \~ View of The River Styx (RLC) \[Love's Oni\]](https://osu.ppy.sh/beatmapsets/140944#osu/351752)**

Reviewer 2: ```suggestion
  - **[Demetori - Higan Retour \~ View of The River Styx (RLC) \[Love's Oni\]](https://osu.ppy.sh/beatmapsets/140944#taiko/351752)**
```

4. Results will be published via Statistics site.

### Mappool instructions

1. There will be 1 mappool for the Group Stage, 1 mappool for Round 16, 1 mappool for the Quarter-finals, 1 mappool for the Semi-finals and 1 mappool for the Finals.
2. Each mappool consists of 5 brackets: NoMod, [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers), [DoubleTime](/wiki/Game_Modifiers) and FreeMod
3. Each mappool consists of 23 maps in total.
4. Each mappool has one tiebreaker
5. The NoMod bracket will be played with no modes activated.
6. The Hidden, HardRock and DoubleTime bracket will be played with the respective modes activated.
7. The FreeMod bracket will have FreeMod activated. Every individual player can pick Hidden, HardRock or no mod at all. Players **must not** select more than one mod.
   - When playing a FreeMod map, at least one player of each team must have one mode activated.
8. The tiebreaker will be played under FreeMod conditions.
   - When playing the tiebreaker, no one needs to have a mode activated.
9. The size of the NoMod bracket will be:
   - 10 in Group Stage
   - 6 in Knock-Out Stages
10. The size of the mod-specific brackets will be:

>    - 3 in Group Stage

Reviewer 2: ```suggestion
    - 3 in Group Stage
```

>    - 4 in Knock-Out Stages

Reviewer 2: ```suggestion
    - 4 in Knock-Out Stages
```

4. Results will be published via Statistics site.

### Mappool instructions

1. There will be 1 mappool for the Group Stage, 1 mappool for Round 16, 1 mappool for the Quarter-finals, 1 mappool for the Semi-finals and 1 mappool for the Finals.
2. Each mappool consists of 5 brackets: NoMod, [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers), [DoubleTime](/wiki/Game_Modifiers) and FreeMod
3. Each mappool consists of 23 maps in total.
4. Each mappool has one tiebreaker
5. The NoMod bracket will be played with no modes activated.
6. The Hidden, HardRock and DoubleTime bracket will be played with the respective modes activated.
7. The FreeMod bracket will have FreeMod activated. Every individual player can pick Hidden, HardRock or no mod at all. Players **must not** select more than one mod.
   - When playing a FreeMod map, at least one player of each team must have one mode activated.
8. The tiebreaker will be played under FreeMod conditions.
   - When playing the tiebreaker, no one needs to have a mode activated.
9. The size of the NoMod bracket will be:
   - 10 in Group Stage
   - 6 in Knock-Out Stages
10. The size of the mod-specific brackets will be:

>    - 3 in Group Stage

Reviewer 2: Broken indent.
